lib/headquarters/api/endpoints.rb

```diff
@@ -0,0 +1,12 @@
+module Headquarters
+  class API
+    module Endpoints
```

Comment:
Monkey patching a class to add modules seems odd to me. Why doing it like this and not creating modules that are then included in the class?